README.md

```diff
@@ -63,15 +63,15 @@ You must either construct an `o-tooltip` object or fire an `o.DOMContentLoaded`
 #### Constructing an o-tooltip
 If you have setup your tooltip declaratively, the following applies:
 ```js
-const oTooltip = require('o-tooltip');
+const Tooltip = require('o-tooltip');
 let tooltipEl = Document.getElementById('my-tooltip-element');
-const oTooltip = new oTooltip(tooltipEl, '#declarative-tooltip-target');
+const tooltip = new Tooltip(tooltipEl, '#declarative-tooltip-target');
```

Comment:
I can't see the code that supports this (providing a CSS selector as a second arg), I guess it just needs the first argument?

Reply:
It should be:
`const tooltip = new Tooltip(tooltipEl, {target: '#declarative-tooltip-target'});`

Source - https://github.com/Financial-Times/o-tooltip/blob/master/src/js/tooltip.js#L30

Reply:
As it's documenting the declarative approach should it be `data-o-tooltip-target` in the DOM and this arg blank?

Reply:
Yes 👍